pkg/instance_sharing/types.go

```diff
@@ -4,11 +4,21 @@ const (
 	ReferencePlanDescription = "Allows to create a reference to a shared service instance from any environment in a subaccount and manage service bindings to that service instance."
 	ReferencePlanName        = "reference-instance"
 
-	ReferencedInstanceIDKey = "referenced_instance_id"
-
 	SupportsInstanceSharingKey = "supportsInstanceSharing"
 
-	ReferencePlanNameSelector     = "plan_name_selector"
-	ReferenceInstanceNameSelector = "instance_name_selector"
-	ReferenceLabelSelector        = "instance_labels_selector"
+	ReferencedInstanceIDKey         = "referenced_instance_id"
+	ReferencedInstanceIDTitle       = "Reference Instance ID"
+	ReferencedInstanceIDDescription = "Specify the ID of the instance to which you want to refer."
+
+	ReferenceInstanceNameSelectorKey         = "instance_name_selector"
+	ReferenceInstanceNameSelectorTitle       = "Find by instance name"
+	ReferenceInstanceNameSelectorDescription = "You can use various search criteria to find shared instances to which you want to create a reference."
```

Comment:
This isn't what we agreed on.
It should be:
Create a reference to a shared instance that has the name you specified.

Reply:
You can use the instance name to find a shared instance to which you want to create a reference.